### PR TITLE
[corona_stats] Add Mattermost support

### LINKED ISSUE #150: Support corona_stats command-set on mattermost

### LINKED ISSUE #154: The messages which has "add command-set to your Slack with Commander" should only be printed if the client is slack
We don't want to show this message for non-slack clients. For non-slack clients, for now we don't have to print anything

### PR DESCRIPTION
**Pull request by bhageena**
Desktop:
Screenshot from 2020-04-22 23-14-43

Mobile:
Screenshot from 2020-04-22 23-04-28

Closes #150

### CODE REVIEW

**Review by niks3089 — COMMENTED** [review]
The emoji doesn’t work with teams unfortunately. Can we remove that?

File: corona_stats/README.md [hunk 1]
```diff
@@ -22,5 +22,3 @@ Available commands:
 `/nc corona_stats <Country Name | Abbreviation > -r <State Name | Abbreviation>`  -- to get stats for a specific state.
 
 `/nc corona_stats <Country Name | Abbreviation > -r <District Name>`  -- to get stats for a specific district.
-
-[Read More](https://nimbella.com/blog/get-live-coronavirus-stats-in-slack-with-nimbella-commander/)
```
Comment (satyarohith) [inline corona_stats/README.md:26]: I think it's a good thing to keep a link to our blog post. We can reframe it to something like "Read our blog on corona stats in Slack to learn more."

Comment (bhageena) [reply corona_stats/README.md:26]: thanks, added it. I thought it to be slack specific and wanted to keep the readme generic. Further evolution will fix things.

**Comment by bhageena** [comment]
> The emoji doesn’t work with teams unfortunately. Can we remove that?
Replaced the emoji text, it should work for teams as well.

File: corona_stats/README.md [hunk 2]
```diff
@@ -1,9 +1,7 @@
-# Get live Coronavirus stats in Slack with Nimbella Commander
+# Get live Coronavirus stats in Slack/Mattermost with Nimbella Commander
```
Comment (niks3089) [inline corona_stats/README.md]:
```suggestion
# Get live Coronavirus stats with Nimbella Commander
```

**Review by niks3089 — APPROVED** [review]

**Pull request merged**